hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -8,14 +8,285 @@ title: "OTP Controller Technical Specification"
+They are **NOT** directly sourced from the OTP macro itself.
+Thus the security of both volatile (OTP controller) and non-volatile (OTP IP) storage becomes important.
+
+### Collateral
+
+The OTP controller for OpenTitan contains the five logical partitions shown below.
+
+Partititon     | Collateral                 | Secret | Buffered | WR Lockable  | RD Lockable  | Usage
+---------------|----------------------------|--------|----------|--------------|--------------|----------------------------------------------------------------
+CREATOR_SW_CFG | Device manufacturing data  | no     | no       | yes (digest) |   yes (CSR)  | Device-specific calibration (Clock, LDO, RNG, device identity).
+OWNER_SW_CFG   | Software configuration     | no     | no       | yes (digest) |   yes (CSR)  | Data that changes software behavior, specifically in the ROM. <br> E.g., enabling defensive features in ROM or selecting failure modes if verification fails.
+HW_CFG         | Hardware configuration     | no     | yes      | yes (digest) |      no      | Hardware configuration bits used to hardwire specific hardware functionality. <br> E.g., raw entropy accessibility or flash scrambling bypass range.
+LIFE_CYCLE     | Life-cycle state, counters | no     | yes      |     no       |      no      | Life-cycle related bits (ADD link). **Note**, this partition cannot be locked as the life-cycle state needs to be able to advance to RMA in the field.
+SECRET         | Tokens and Keys            | yes    | yes      | yes (digest) | yes (digest) | Device-unique secrets.
+
+Generally speaking, the production life cycle of a device is split into 5 stages "Manufacturing" -> "Calibration and Testing" -> "Provisioning" -> "Mission" -> "RMA".
+OTP values are usually programmed during "Calibration and Testing", "Provisioning" and "RMA" stages, as explained below.
+A detailed listing of all the collateral and the corresponding memory map can be found in the Programmer's Guide Section further below.
+
+### Calibration and Test
+
+During this stage, the device is tested for functionality and calibrated to ensure uniformity.
+The calibration can focus on a number of things, but usually is centered around adjusting clock, voltage and timing sources to remove process variation.
+These calibration values are programmed into the CREATOR_SW_CFG partition, as they are non-secret values meant to be read out by software and programmed into respective peripherals.
+
+Early on during this stage, the various secret unlock tokens are also programmed into the secret partition and harvested by the silicon creator.
+
+### Provisioning
+
+During this stage, the device is provisioned with the final firmware and a "unique" seed or identity.
+The secret partition is populated with root secrets and keys that are critical to establishing the device identity.
+
+As part of injecting the final firmware, the SKU-specific hardware and software configurations are also programmed.
+
```

Comment:
@moidx 
@zi-v 
@cdgori 
@gkelly 

it would be good if you guys can review these two sections `Calibration and Test` and `Provisioning` and make sure it is a reasonable representation.  This text here is just meant to be give the reader a sample, but does not need to be perfectly accurate.